modules/jjui/hm.nix

```
@@ -0,0 +1,137 @@
+{ mkTarget, lib, ... }:
+mkTarget {
+  config =
+    { colors, polarity }:
+    {
+      programs.jjui.settings.ui.colors = with colors.withHashtag; {
+        # Original template author: Victor Borja <vborja@apache.org> (https://github.com/vic)
+        # See https://github.com/vic/tinted-jjui
+
+        text.fg = base05;
+        text.bg = base00;
+        dimmed.fg = base03;
+        dimmed.bg = base00;
+        title.fg = base0D;
+        title.bold = true;
+        shortcut.fg = base0E;
+        matched.fg = base0A;
+        border.fg = base03;
+        selected = {
+          fg = base05;
+          bg = base01;
+          bold = true;
+        };
+
+        source_marker = {
+          fg = base00;
+          bg = base0C;
+          bold = true;
+        };
+        target_marker = {
+          fg = base00;
+          bg = base0B;
+          bold = true;
+        };
+
+        status.bg = base01;
+        "status title" = {
+          fg = base00;
+          bg = base0D;
+          bold = true;
+        };
+
+        "revset title" = {
+          fg = base0D;
+          bg = lib.mkIf (polarity == "light") base01;
+          bold = true;
+        };
+        "revset text".fg = base05;
+        "revset text".bold = true;
+        "revset completion text".fg = base05;
+        "revset completion matched".fg = base0A;
+        "revset completion matched".bold = true;
+        "revset completion dimmed".fg = base03;
+        "revset completion selected".bg =
+          if polarity == "light" then base06 else base02;
```

note that polarity isn't always accurate. further this assumes that `"either"` should be treated as dark, which i think the opposite should be assumed: 
```suggestion
          if polarity == "dark" then base02 else base06;
```